OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -92,7 +92,26 @@ public function __construct(HttpClientInterface $httpClient, HttpUtils $httpUtil
 
         // Resolve merged options
         $resolver = new OptionsResolver();
+
         $this->configureOptions($resolver);
+
+        // See Symfony\Component\Form\ResolvedFormType logic
+        $baseClasses = array(
+            'HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\AbstractResourceOwner',
+            'HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\GenericOAuth1ResourceOwner',
+            'HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\GenericOAuth2ResourceOwner',
```

Comment:
If I remove it, all resource owners extending `GenericOAuth1ResourceOwner` and `GenericOAuth2ResourceOwner` will not get deprecation error on running. This is why I did that.

Same argument for: https://github.com/hwi/HWIOAuthBundle/pull/808#discussion_r31998366